kernel/src/hil/symmetric_encryption.rs

```diff
@@ -0,0 +1,16 @@
+use returncode::ReturnCode;
+
+pub trait SymmetricEncryptionDriver {
+    fn init(&self);
+    fn set_key(&self, key: &'static mut [u8]);
+    // fn encrypt(&self, plaintext: &'static mut [u8], len: u8);
```

Comment:
You can use rust's [unimplemented!()](https://doc.rust-lang.org/core/macro.unimplemented.html) for this, i.e.

```rust
impl SymmetricEncryptionDriver {
    fn encrypt(&self, plaintext: &'static mut [8], len: u8) {
        unimplemented!();
    }
}
```

and rustc will do the right things - not throw unused warnings, panic! if that's hit